src/tests/processing/test_steps.py

```
@@ -1,24 +1,26 @@
 import pytest
+
+from openparse import consts
 from openparse.processing import (
-    ProcessingStep,
     RemoveTextInsideTables,
     RemoveRepeatedElements,
     RemoveFullPageStubs,
     CombineNodesSpatially,
     CombineBullets,
     CombineHeadingsWithClosestText,
+    CombineNodesSpatially,
```

Ruff (0.11.9)

Redefinition of unused `CombineNodesSpatially` from line 8

Remove definition: `CombineNodesSpatially`

(F811)